mollie/api/objects/capture.py

```diff
@@ -0,0 +1,40 @@
+from .base import Base
+
+
+class Capture(Base):
+    @classmethod
+    def get_resource_class(cls, client):
+        from ..resources.captures import Captures
+        return Captures(client)
+
+    @property
+    def id(self):
+        return self._get_property('id')
+
+    @property
+    def mode(self):
+        return self._get_property('mode')
```

Comment:
This property is not under test, see https://travis-ci.org/mollie/mollie-api-python/jobs/535273819